java/impl/src/test/java/net/sf/mmm/code/impl/java/JavaExtendedContextWithMavenAndSourceCodeTest.java

```diff
@@ -86,7 +86,8 @@ public void testContextFromOwnMavenProject() {
 
     // then
     assertThat(type.getContext()).isSameAs(context);
-    assertThat(type.getDoc().getLines()).contains("Abstract top-level interface for any item of code as defined by this API. It reflects code structure.");
+    assertThat(type.getDoc().getLines()).contains(
+        "Abstract top-level interface for any item of code as defined by this API. It reflects code structure.");
```

Comment:
is there any real change in this file? I only see that you are using a different line-length in the formatter and therefore all changes are just formatter chnages in this file...